### ISSUE TITLE
Bundle Windows download with required DLLs?

### ISSUE THREAD

**Issue by gitwah** [bcpierce00/unison#186]
Hello! 
First of all, thanks for unison, I have been using it for quite some time on my linux machine! Recently I wanted to set it up on my parents' windows computer.
After downloading it and starting the .exe the program exits telling me it is missing some .dll. I don't know that much about windows but just downloading random .dll files from all over the web didn't seem like a good idea. My google-fu wasn't sufficient to find any other solution.
So basically what I am suggesting is: Could you include the missing .dll files in the download for the windows version / tell me why that would be a bad idea?

**Comment by yannickmodahgouez** [comment]
Could you provide a more detailed error message ?

**Comment by gitwah** [comment]
"This application has failed to start because libgtk-win32-2.0-0.dll was not found. Re-installing the application may fix this problem."

I remember more than one .dll-files missing, though. After getting one .dll and placing it in the right directory the error message returned requesting another dll file

**Comment by JackeJR** [comment]
on the same note, I think the instructions for compiling for windows is quite outdated.
compiling with mingw is doable but I have never been successful with msvc.

**Comment by qouoq** [comment]
I happen to stumble on the same problem today too.

Following the instructions provided on the [Unison wiki](https://alliance.seas.upenn.edu/~bcpierce/wiki/index.php?n=Main.UnisonFAQOSSpecific#windows), installing the [GTK+ for Windows Runtime](https://gtk-win.sourceforge.io/home/index.php/Main/Downloads) should install all the necessary libraries.

**Comment by yannickmodahgouez** [comment]
@gitwah I assume you're using the graphical interface. Did you try the text interface ?

@JackeJR I just compiled the text version using Visual studio 2017 Community. Here is a PR that helped making it work : https://github.com/bcpierce00/unison/pull/195 and here is the binary : https://github.com/sticksh/unison/releases.
I agree that the windows documentation needs a rewrite.

**Comment by PeterMosmans** [comment]
Slightly related: Here are some adventures on how to compile Unison on Windows using msys2 / mingw64: https://www.onwebsecurity.com/configuration/its-about-the-journey-compiling-64-bit-unison-gtk2-on-windows.html

**Comment by gdt** [comment]
There is work underway to have a reliable windows build recipe which will supercede this request.

**Comment by MelleNi** [comment]
Trying to run the latest unison-gui.exe I just downloaded, I am missing:
-libcario-2.dll
-libfreetype-6.dll
-libfontconfig-1.dll
-libgdk-3-0.dll

Why are these files not bundled in the release? Why is there no info about this anywhere on the github (as far as I can find)?

**Comment by tleedjarv** [comment]
Take the DLLs from the 2.53.0 release.
